Resources/public/js/views/fields/ez-float-editview.js

```diff
@@ -13,7 +13,7 @@ YUI.add('ez-float-editview', function (Y) {
 
     var L = Y.Lang,
         FIELDTYPE_IDENTIFIER = 'ezfloat',
-        FLOAT_PATTERN = "\\-?\\d*\\.?\\d+"; // WARNING: each backslash is doubled, because it is escaped on output otherwise
```

Comment:
as far as I can tell, that's not needed, we don't rely on the form not being submitted when an input is invalid and we don't use the default message as well so unless we have other(s) issue(s), fixing the pattern is the best way to fix the issue as it's the root of it.